src/templates/actors/partials/character-header.html

```diff
@@ -50,48 +52,52 @@ <h1 class="charname">
       <label>{{localize 'OSE.details.title'}}</label>
     </li>
     {{/if}}
-    <li>
-      <input
-        type="text"
-        name="system.details.alignment"
-        value="{{system.details.alignment}}"
-        data-dtype="String"
-      />
-      <label>{{localize 'OSE.details.alignment'}}</label>
-    </li>
-  </ul>
-  <ul class="summary flexrow">
-    <li class="flex3">
-      <input
-        type="text"
-        name="system.details.class"
-        value="{{system.details.class}}"
-        data-dtype="String"
-      />
-      <label>{{localize 'OSE.details.class'}}</label>
-    </li>
-    <li
-      class="{{#if (gte system.details.xp.value system.details.xp.next)}}notify{{/if}}"
-    >
-      <input
-        type="text"
-        name="system.details.level"
-        value="{{system.details.level}}"
-        data-dtype="Number"
-      />
-      <label>{{localize 'OSE.details.level'}}</label>
-    </li>
-    <li class="flex2">
-      <input
-        type="text"
-        name="system.details.xp.value"
-        value="{{system.details.xp.value}}"
-        data-dtype="Number"
-      />
-      <label>{{localize 'OSE.details.experience.base'}}</label>
-      {{#if system.details.xp.bonus}}
-      <span class="xp-bonus">+{{system.details.xp.bonus}}%</span>
-      {{/if}}
-    </li>
+    {{#if isOwnerOrObserver}}
+      <li>
+        <input
```

Comment:
An observer probably should not see the `input` fields (to be able to edit the values - even though they would error if they tried). Thoughts on either displaying the `input` or just the raw value depending on whether they are an owner or observer?